extensions/rendering-hints/README.md

```diff
@@ -0,0 +1,63 @@
+# Rendering Hints Extension Specification
+
+- **Title: Rendering Hints**
+- **Identifier: rendering_hints**
+- **Field Name Prefix: render**
+- **Scope: Item**
+- **Extension [Maturity Classification](../README.md#extension-maturity): Proposal**
+
+One of the things many people want to do with data cataloged in STAC is to render it, often doing so directly
+from the assets (usually [Cloud Optimized GeoTIFF's](http://cogeo.org)) with dynamic web tile services. To 
+perform well these services benefit from a few 'hints' that can help the renderer perform more effectively. 
+
+This document explains the fields of the STAC Rendering Hints (`render`) Extension to a STAC Item. They are meant
+to be 'hints' - things that could be discovered by inspecting the data, but provided in STAC to help out tools
+that can take advantage of them. It is written with geospatial raster data in mind, but could apply to other data types.
+
+The field names defined herein should be added as fields in the Item Properties object. When specified on an Item, 
+the values are assumed to apply to all non-thumbnail Assets in that Item. 
+
+## Examples
+
+- TODO
+
+## Schema
+
+- TODO
+
+## Item Properties fields
+
+| Field Name       | Type                     | Description |
+| ---------------- | ------------------------ | ----------- |
+| render:max_zoom  | integer | **REQUIRED.** The maximum [zoom level](https://wiki.openstreetmap.org/wiki/Zoom_levels), expressed as an integer, that corresponds with the highest resolution of the data. Specified as [Web Mercator](https://en.wikipedia.org/wiki/Web_Mercator_projection) zoom levels. |
+| render:min_zoom  | integer  | The minimum [zoom level](https://wiki.openstreetmap.org/wiki/Zoom_levels), expressed as an integer, that tells renderers where the overviews of the data stop. |
```

Comment:
I think it would be good to specify whether `min_zoom` and `max_zoom` are inclusive of the given integer.

I think it would be good to define more clearly how `min_zoom` and `max_zoom` are defined:

- At least in the sense of COGs where you're generally creating a raster, what pixel resolution underlies these min/max zoom estimations? Most raster map data is served as either 256x256px tiles or 512x512px tiles. That difference changes the `zoom` level by 1.
- Specify that these are generated by considering the pixel resolution per meter and that for Web Mercator this is technically only valid at the equator.
- If min/max zoom must be an integer, how does rounding happen? Is `max_zoom` the integer zoom beyond which you can't get any more resolution and `min_zoom` the integer zoom before which you can't get any less resolution from overviews? So if these correspond to fractional zooms of 7.5 to 10.5, the min/max zooms would be stored as 7 and 11, respectively? I'm guessing that storing these numbers as integers will be more accessible to more people.

Reply:
- If min/max zooms are a single number for all bands, does the number represent the most detailed band?
- If a less-detailed band had the same number of overviews as a more-detailed band, the less-detailed band would have a smaller `min_zoom` than the more-detailed band